x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/job_creator.ts

```diff
@@ -57,26 +59,54 @@ export class JobCreator {
     stop: boolean;
   } = { stop: false };
 
+  private _unsubscribeAll = new Subject();
+  private modelMemoryEstimator: ModelMemoryEstimator;
+
   constructor(
-    indexPattern: IndexPattern,
-    savedSearch: SavedSearchSavedObject | null,
-    query: object
+    protected _indexPattern: IndexPattern,
```

Comment:
for consistency it would be neater to keep all the declarations at the top of the class.

Reply:
Rolled back these changes